src/caffe/proto/caffe.proto

```diff
@@ -448,6 +448,7 @@ message TransformationParameter {
 message LossParameter {
   // If specified, ignore instances with the given label.
   optional int32 ignore_label = 1;
+  optional int32 only_care_label = 4;
```

Comment:
Same as above: let's consider `select_label` or `specify_label` instead. Also, a word of a comment above would be nice.

Reply:
Thank you very much for your advice. I will modify them.
I am sorry for being late.

Reply:
I will make a new request name as "Selecting only a specified class for backprop using SoftmaxWithLoss #6253"